examples/siri_exm_SX/exx_situationExchange_response.xml

```diff
@@ -35,12 +38,14 @@
      <QualityIndex>certain</QualityIndex>
      <ValidityPeriod>
       <StartTime>2001-12-17T09:30:47.0Z</StartTime>
+      <EndTime>2001-12-17T10:30:47.0Z</EndTime>
      </ValidityPeriod>
-     <MiscellaneousReason>bombExplosion</MiscellaneousReason>
+     <TpegReason>bombExplosion</TpegReason>
      <Severity>severe</Severity>
+     <Priority>3</Priority>
      <Audience>public</Audience>
      <ReportType>point</ReportType>
-     <Summary overridden="true">Bomb at Barchester station</Summary>
+     <Summary overridden="true" xml:lang="EN">Bomb at Barchester station</Summary>
      <Description overridden="true" xml:lang="EN">Building evacuated. AVoid station untile further notice</Description>
```

Comment:
Please remove the "e" from "untile" and replace "furtehr" by "further".
And "AVoid" should probably be "Avoid".

Reply:
Thanks, resolved this!